contents/founders/actual-marketing-budget.md

```diff
@@ -0,0 +1,97 @@
+---
+date: 2023-11-23
+title: How we spend our marketing budget at PostHog - with actual $ figures
+author:
+  - charles-cook
+featuredImage: ../images/blog/transparency.jpeg
+featuredImageType: full
+tags:
+  - Founders
+  - Marketing
+---
+
+It really annoys me that no one is willing to talk in any level of detail about how much they actually spend on marketing in startup-land. 
+
+When I’m trying to figure out what a sensible amount is for PostHog to spend, all I can find are articles that talk very generally about marketing budgets in terms of things like % of revenue or general benchmarks. Which is nice, but [the ranges given are wild](https://www.datadab.com/blog/how-much-saas-companies-spend-on-marketing/) and I want _cold, hard numbers_. 
+
+How much does a typical startup [like PostHog](/about) spend on marketing? And why doesn’t anyone talk about it?
+
+“But Charles, you naive idiot” asked nobody, “surely competitors will use this information against you?”
+
+Um, sure I guess? But especially if you’re into product-led growth (like we are), how you spend marketing $ seems pretty far down the list of competitive advantages. We already [tell everyone](/roadmap) what we’re going to build next, and that doesn’t seem to have hurt us so far.
+
+By sharing some of this info, I’m hoping it’ll help others as a data point. **I am not saying this is the best way to spend your marketing budget.** I have no idea! That’s the point!! But I’m hoping _maybe_ someone will see this and tell us how we could do better...
+
+Let’s get into it. 
```

Comment:
```suggestion
How much does a typical startup [like PostHog](/about) spend on marketing? And why doesn’t anyone talk about it?

I've searched high and the low the best I can find are articles that talk _very generally_ about marketing budgets in terms of things like % of revenue, or general benchmarks. But [the ranges given are wild](https://www.datadab.com/blog/how-much-saas-companies-spend-on-marketing/) and I want _cold, hard numbers_. It's really annoying.

“But Charles, you naive idiot” asked nobody, “surely competitors will use this information against you?”

Um, sure I guess? But, especially if you’re into product-led growth (like we are), how you spend marketing $ seems pretty far down the list of competitive advantages. We already [tell everyone](/roadmap) what we’re going to build next, and that doesn’t seem to have hurt us so far.

By sharing some of this info, I’m hoping it’ll help others as a data point. **I am not saying this is the best way to spend your marketing budget.** I have no idea! That’s the point!! But I’m hoping _maybe_ someone will see this and tell us how we could do better...

Let’s get into it. 
```

Reply:
+1 to Ian's comment about the opening sentence. My suggestion as a revision. Feel to free to go another way.